### DISCUSSION TITLE
Lsp Uninstaller missing

### DISCUSSION THREAD

**Discussion by tonmoy998** [NvChad/NvChad#2469] [General]
![2023-10-31T00:03:09,283753213+06:00](https://github.com/NvChad/NvChad/assets/74363531/e2bdbd63-d999-46a1-bf24-07dfe44c6600)
![2023-10-31T00:03:16,281928705+06:00](https://github.com/NvChad/NvChad/assets/74363531/ab306356-bd72-4add-ac16-246c0851869f)

**Comment by siduck** [comment]
@tonmoy998 only few commands are lazyloaded, just override their plugin cmds in custom plugins
